lib/getHttpIpfsLink.ts

```diff
@@ -0,0 +1,8 @@
+const getHttpIpfsLink = (ipfsLink) => {
+  if (!ipfsLink) return ""
+  const gateway = "https://ipfs.io/ipfs/"
```

Comment:
we already have `lib/getIpfsLink` right? 

do we need another util to convert a CID to a valid gateway url?